com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Adjustment/ReplaceColorNode.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class ReplaceColorNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "ReplaceColor",
+@"
+{
+    Distance = distance(From, In);
+    Out = lerp(To, In, saturate((Distance - Range) / max(Fuzziness, 1e-5f)));
```

Comment:
Since the Distance variable is only used once, how about if we get rid of it and replace Distance with distance(From, In) ?